src/symbols_macos.cpp

```diff
@@ -14,6 +14,19 @@
 #include "symbols.h"
 #include "log.h"
 
+UnloadProtection::UnloadProtection(CodeCache *cc) {
```

Comment:
I don't see its usage in `Symbols::parseLibraries` on macOS

Reply:
5e10cdd462942f2fda16317a8e729b6a19207150